code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -209,9 +209,9 @@ obj/item/asteroid/basilisk_hide/New()
 	if(!alerted)
 		alerted = 1
 		spawn(chase_time)
-			if(alerted)
+			var/turf/T = get_turf(src)
+			if(alerted && !isspace(T))
```

Comment:
I believe this macro is intended to be used with areas, not turfs, because it checks to see if the compared thing is strictly type `/area` as per macros.dm L314

Unless there's another macro I don't know about

Reply:
So space _does_ have an area, huh. Yeah this is wrong